app/controllers/admin/words_controller.rb

```diff
@@ -50,7 +75,7 @@ def destroy
 
   private
   def word_params
-   params.require(:word).permit(:name, choices_attributes: [:content])
+   params.require(:word).permit(:name, choices_attributes: [:content, :correct_ans])
```

Comment:
This is correct. Putting the **:correct_ans** attribute to save the choice answer is a good way to save it when making the word & choice at the same time.